logging-utils/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "google-logging-utils",
+  "version": "0.0.2",
+  "description": "An ad-hoc debug logger package for other libraries",
+  "main": "build/src/index.js",
+  "files": [
+    "build/src"
+  ],
+  "scripts": {
+    "pretest": "npm run prepare",
+    "test": "c8 mocha build/test",
+    "lint": "gts check test src samples",
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r test/fixtures build/test",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "precompile": "gts clean",
+    "samples-test": "echo no samples 🙀",
```

Comment:
There is a sample, should there either be a test or should this actually say no samples tests?

Reply:
@sofisl wanted to add a system test of some kind, so maybe a sample test would be the best place.

I don't have an official sample tag, and I don't know if we even want one, so I made one up as a placeholder. Any thoughts on that?

Reply:
For now I'm just "commenting out" the snippet tags. We'll need to revisit that and/or update the typeless bot later to make automatic JS sample updates possible.